packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -770,19 +770,22 @@ const UpdateAction: DocumentActionComponent = ({
       setSubmitting(true);
 
       try {
-        const { errors } = await validate();
-
-        if (errors) {
-          toggleNotification({
-            type: 'danger',
-            message: formatMessage({
-              id: 'content-manager.validation.error',
-              defaultMessage:
-                'There are validation errors in your document. Please fix them before saving.',
-            }),
-          });
-
-          return;
+        // Don't run the validation for drafts
+        if (activeTab !== 'draft') {
```

Comment:
This is not what we should do. We should just run some validation and skip others instead.